src/components/header/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
+import styles from './header.module.css';
+import { auth } from '@/modules/authentication';
+import SignOutButton from '../sign-out';
+
+export default async function Header() {
+  const session = await auth();
+  return (
+    <header className={styles.header}>
+      <nav className={styles.nav}>
+        <ul>
+          <li>
+            <Link href="/">Driftsportal</Link>
+          </li>
+        </ul>
+      </nav>
+      <div className={styles.auth}>
+        {session?.user && <p>Du er logget inn som {session.user.name}</p>}
+        {session?.user && <SignOutButton />}
+      </div>
```

Comment:
Don't accept the suggestion since the formatting isn't correct, but couldn't we do something like this instead?

```suggestion
      {session?.user && (
        <div className={styles.auth}>
          <p>Du er logget inn som {session.user.name}</p>
          <SignOutButton />
      </div>)}
```

Reply:
Principally agreed with Adrian here, but also, we'll probably change this up a lot when design is ready so I don't think we need to be as "correct" here.

Reply:
As Mikael says, this was just a proof of concept :)